src/Database/migrations/2023_05_02_160733_create_user_disks_table.php

```diff
@@ -17,7 +17,7 @@ public function up()
             $table->id();
             $table->timestamps();
             $table->timestamp('expires_at')->index();
-            $table->string('name', 128)->unique();
+            $table->string('name', 128);
```

Comment:
This doesn't work. Migrations can never be retroactively changed (because they are already applied in production). Instead, you have to create a new migration that changes the previous one.

In the new migration you have to [drop the previous unique index](https://laravel.com/docs/10.x/migrations#dropping-indexes) with `$table->dropUnique('user_disks_name_unique')`. Then you can create a new compound unique index with `$table->unique(['name', 'user_id'])`. The compound index will make sure that no user can have two disks with the same name on the database level. However, you still need the new validation to show a graceful error to the user.